src/llv8-constants.cc

```diff
@@ -537,7 +562,8 @@ void Types::Load() {
   kOddballType = LoadConstant("type_Oddball__ODDBALL_TYPE");
   kJSObjectType = LoadConstant("type_JSObject__JS_OBJECT_TYPE");
   kJSAPIObjectType = LoadConstant("APIObjectType");
-  kJSSpecialAPIObjectType = LoadConstant("APISpecialObjectType");
+  kJSSpecialAPIObjectType = LoadConstant("SpecialAPIObjectType",
+                                         "APISpecialObjectType");
```

Comment:
I think this has always been called `SpecialAPIObjectType`.  `APISpecialObjectType` may have been a typo.

@indutny Can you confirm?